MagaluCloud/magalu
# feat(tf): add Connection Resources

## Description

This PR adds support for a second way of defining Action Resources via OpenAPI links. To do so, one must choose an action path (for example, attach a block storage volume to a virtual machine instance) and add links to it:
- `read` -> a request that informs TF if the action has taken place. Can/should have a `wait-termination` flag for validation
- `delete` -> request to detach

## Related Issues

- Closes #421

### Pull request checklist


## How to test it

Change the current `mgc_block-storage_volume_attach-to-vm` resource name in the `main.tf` file to `mgc_block-storage_volume_attach`, to use the new action resource. Test it out normally, create it, delete it, delete it from outside and see if TF notices a problem, etc... Happy AND unhappy paths
